manim/mobject/coordinate_systems.py

```diff
@@ -143,7 +143,7 @@ class Axes(VGroup, CoordinateSystem):
     }
 
     def __init__(self, **kwargs):
-        CoordinateSystem.__init__(self, **kwargs)
+        CoordinateSystem.__init__(self)
```

Comment:
```suggestion
        CoordinateSystem.__init__(self, kwargs.get('dim', 2))
```

or something like that

Reply:
https://github.com/ManimCommunity/manim/blob/19d5563747497ee51dad6a6f5c48597bdd74b0a8/manim/mobject/coordinate_systems.py#L27-L33

It's a default param.

Reply:
Sorry, I should have been more clear. What I'm trying to say is that `Axes` should accept a `dim` parameter as a kwarg and pass it along to `CoordinateSystem` if it is present.

Reply:
So you mean instead of having `NumberLine`, `Axes` and `ThreeDAxes`, we should have a single `Axes` class with modifiable dimension?

Reply:
Oh oh oh sorry I didn't realize that it was already split by class. Never mind me.

Reply:
I would still make it explicity and pass `dim=2` but whatever.